Make packages window responsive

ederin opened the pull request:
Width of "From name" and "To name" columns is adjusted when window resize event is emitted, so packages table always fits packages window.

Resolves: #14

PJ-Finlay commented:
Thanks! Testing now

PJ-Finlay commented:
Thanks for making this pull request! This is definitely the right way to do things over locking the window from resizing. Can you make some small changes before this gets merged? 
1. On my setup resizing works fine but when the window is first opened the "From name" label gets cut off.
2. Less important than the previous one but when the window scales down is it possible to not scale with empty table space but with just the background color?

Great work!

![image](https://user-images.githubusercontent.com/13067016/102946773-3ecf9f80-4487-11eb-87e9-ace96ec949d2.png)
![image](https://user-images.githubusercontent.com/13067016/102946806-56a72380-4487-11eb-99e9-2694734a7de7.png)

ederin commented:
Hi,

thank you for your review of this pull request.

> 1. On my setup resizing works fine but when the window is first opened the "From name" label gets cut off.

It seems that `resizeColumnsToContents()` sometimes doesn't work properly on columns when `ResizeMode == Stretch`. I'm not sure why is it happening. I added padding to right side of affected labels and aligned them to left instead. It has nice side effect that it also limits minimal size of packages window. Maybe this padding should be later refactored into separate QSS file...

> 2. Less important than the previous one but when the window scales down is it possible to not scale with empty table space but with just the background color?

I added `self.layout.addStretch()` to try it out. Yeah, it looks better without that white table space.

PJ-Finlay commented:
Looks good merging!
